documentation/book/proc-deploying-example-clients.adoc

```diff
@@ -1,26 +1,29 @@
 // Module included in the following assemblies:
 //
-// assembly-hello-world.adoc
+// getting-started.adoc
 
 [id='deploying-example-clients-{context}']
-== Deploying example clients
+= Deploying example clients
+
+This example provides instructions for deploying a client to communicate with the Kafka cluster.
 
 .Prerequisites
 * An existing Kafka cluster for the client to connect to.
 
 .Procedure
 
-You can deploy the client in a single command.
-This is an example command you should substitute _test_ for your topic and _my-cluster-kafka-bootstrap:9092_ by your server address for both producer and consumer.
-
 . Deploy the producer
 +
-`oc run kafka-producer -ti --image={DockerKafka} --restart=Never \-- bin/kafka-console-producer.sh --broker-list _my-cluster-kafka-bootstrap:9092_ --topic _test_`
+[source,subs="+quotes,attributes"]
+oc run kafka-producer -ti --image={DockerKafka} --restart=Never \-- bin/kafka-console-producer.sh --broker-list __<my-producer>__:__<my-port>__ --topic _<my-topic>_
+
+. Type your message into the console where the producer is running
 
-. Send messages by writing into the console where you run the producer and hit the enter.
+. Press *Enter* to send the message.
 
 . Deploy the consumer
 +
-`oc run kafka-consumer -ti --image={DockerKafka} --restart=Never \-- bin/kafka-console-consumer.sh --bootstrap-server _my-cluster-kafka-bootstrap:9092_ --topic _test_ --from-beginning`
+[source,subs="+quotes,attributes"]
```

Comment:
The same as before why "my-consumer" where should he the bootstrap service?

Reply:
I was trying to make it styled like the replaceable.  Using that style, what should it say? 

I can change:

__<my-consumer>__

TO

__<my-cluster-kafka-bootstrap>__

Does that make sense?

Reply:
Paolo is right. The bootstrap service should be ``_<cluster-name>-kafka-bootstrap:9092_`.

Reply:
All better